scripts/ci/job_hooks/before_e2e.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+npx @alfresco/adf-cli scan-env --host "$E2E_HOST" -u "$E2E_ADMIN_EMAIL_IDENTITY" -p "$E2E_ADMIN_PASSWORD_IDENTITY"
```

Comment:
in the adf pipeline the cli commands needs to be call from node modules otherwise you are not going to get the new changes